pkg/operator/csr/cert_controller.go

```diff
@@ -151,9 +157,16 @@ func (c *clientCertificateController) sync(ctx context.Context, syncCtx factory.
 				Name:      c.SecretName,
```

Comment:
set known anntation values?

Reply:
I'd prefer to contain all metadata changes once in `NeedsTLSMetadataUpdate`